app/views/RoomView/styles.js

```diff
@@ -75,5 +76,16 @@ export default StyleSheet.create({
 		fontSize: 16,
 		fontWeight: '500',
 		color: '#0C0D0F'
+	},
+	scrollToBottomButton: {
+		position: 'absolute',
+		width: 40,
+		height: 40,
+		alignItems: 'center',
+		justifyContent: 'center',
+		right: 30,
```

Comment:
This should be different on iPhone X*.
You should use our custom `deviceInfo.isNotch()` and also test the device orientation.